cmd/handlers.go

```diff
@@ -48,29 +47,33 @@ func init() {
 	serverCmd.PersistentFlags().StringVar(&tlsCertPath, "tls-cert-file", "", "Path to TLS certificate file")
 	serverCmd.PersistentFlags().StringVar(&tlsKeyPath, "tls-key-file", "", "Path to TLS key file")
 	serverCmd.PersistentFlags().StringVar(&rulesFile, "rules-file", "", "File that contains the rules that will be applied for the Kubernetes resources.")
-	serverCmd.PersistentFlags().StringVar(&slackToken, "slack-token", "", "Slack API Token to enable Hammurabi notifications")
+	serverCmd.PersistentFlags().StringVar(&slackToken, "slack-token", "", "Slack API Token to enable Aegir notifications")
 	serverCmd.PersistentFlags().StringVar(&listenPort, "port", "8443", "TCP port that connections will be listen.")
 }
 
 func initConfig() {
 	if rulesFile == "" {
-		fmt.Printf("You must provide a rules file valid path. Eg: %s %s --rules-file=/path/to/file/rules.yaml\n", RootCmd.Name(), serverCmd.Name())
-		os.Exit(1)
+		log.Fatalf("You must provide a rules file valid path. Eg: %s %s --rules-file=/path/to/file/rules.yaml\n", RootCmd.Name(), serverCmd.Name())
 	}
 }
 
 var (
 	universalDeserializer = serializer.NewCodecFactory(runtime.NewScheme()).UniversalDeserializer()
+	skippedNamespaces     []string
 )
 
 type validationFunc func(*v1beta1.AdmissionRequest) []*utils.Violation
 
 func validateRules(req *v1beta1.AdmissionRequest) []*utils.Violation {
+	skippedNamespaces, _ = utils.GetEnvAsSlice("SKIP_NAMESPACES", ",")
```

Comment:
you could initialise when declaring `skippedNamespaces` instead of every time executing `validateRules`.

Reply:
Totally agreed.
Thank you for the amazing review as usual